experimental/origin-dapp2/package.json

```diff
@@ -44,6 +44,7 @@
     "graphql-tag": "^2.10.1",
     "identicon.js": "^2.3.3",
     "ipfs-api": "^26.1.2",
+    "jest-docblock": "^22.2.0",
```

Comment:
This was necessary because babel-plugin-fbt wasn't installing jest-docblock which led to module not found errors. This might have been resolved with the hoisting support for babel-plugin-fbt.

Reply:
Just checked and this is still required even with that hoisting change. 😞